VMEncryption/main/check_util.py

```diff
@@ -326,3 +328,33 @@ def is_non_fatal_precheck_failure(self):
             detected = True
             self.logger.log("PRECHECK: Unsupported mount scheme detected")
         return detected
+
+    def preInitializationCheck(self,logger,imdsStoredResults,iMDSUtil):
+        '''This function is checking the VM compatibility for ADE'''
+        logger.log('Pre initialization check Start.')
```

Comment:
checkUtil class has logger as variable. Use self.logger instead and remove logger from function parameter.

Reply:
done